BingBertSquad/convert_bert_ckpt_to_deepspeed.py

```diff
@@ -0,0 +1,290 @@
+# coding=utf-8
+# This script references to below file from HuggingFace:
+#   https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_bert.py:d541938
```

Comment:
This link just goes to a 404 for me. I think we want this one?
https://github.com/huggingface/transformers/blob/d541938/src/transformers/modeling_bert.py

Reply:
thanks Jeff, will update it with this link.